src/sentry/utils/security/orgauthtoken_token.py

```diff
@@ -0,0 +1,45 @@
+import secrets
+from base64 import b64decode, b64encode
+from datetime import datetime
+
+from django.conf import settings
+
+from sentry.utils import json
+
+SENTRY_ORG_AUTH_TOKEN_PREFIX = "sntrys_"
+
+
+def generate_token(org_slug: str, region_url: str):
+    sentry_url = settings.SENTRY_OPTIONS.get("system.url-prefix")
+    payload = {
+        "iat": datetime.utcnow().timestamp(),
+        "url": sentry_url,
+        "region_url": region_url,
+        "org": org_slug,
+    }
+    secret = b64encode(secrets.token_bytes(nbytes=32)).decode("ascii").rstrip("=")
+
+    json_str = json.dumps(payload)
+    payload_hashed = base64_encode_str(json_str)
+
+    return f"{SENTRY_ORG_AUTH_TOKEN_PREFIX}{payload_hashed}_{secret}"
```

Comment:
```suggestion
    payload_encoded = base64_encode_str(json_str)

    return f"{SENTRY_ORG_AUTH_TOKEN_PREFIX}{payload_encoded}_{secret}"
```

Small nit, but this is _encoded_ not _hashed_.

Reply:
true! :)